tests/test_capture_timing_pipeline.py

```
@@ -105,3 +106,69 @@ def test_visualization_device_frame_is_unwrapped() -> None:
+            is_data_source=True,
+        )
+        for index in range(len(packets))
+    }
+    pipeline = Pipeline.__new__(Pipeline)
+    pipeline.operations = operations
+    pipeline.device_input_uuids = tuple(operations)
+    pipeline.flow_manager = cast(
+        Any,
+        SimpleNamespace(
+            operation_outputs={
+                uuid: packet for uuid, packet in zip(operations, packets)
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)